content/authoring/kata/create-first-kata.md

```diff
@@ -149,7 +149,7 @@ You can check the list of [privileges](/references/gamification/privileges/#priv
 
 The description field is used to provide instructions to the users who will be training on the kata. This field recognizes [Github flavored markdown](/references/markdown/) with a set of [Codewars extensions](/references/markdown/extensions/). You can use the `Preview` tab to see how it will look like when presented to users.
 
-Description of a kata is the first thing others see and decides on the user's first impression. If description is good, users will be probably encouraged took into the kata further. If description is bad, people won't hesitate to point this out, rising issues and downvoting your kata.
+The description of a kata is the first thing others see and decides on the user's first impression. If the description is good, users will be probably encouraged to take the kata further. If the description is bad, people won't hesitate to point this out, rising issues and downvoting your kata.
```

Comment:
```suggestion
The description of a kata is the first thing others see and forms the basis of users' first impressions. If the description is good, users will probably be encouraged to take the kata further. Otherwise, people won't hesitate to raise issues and downvote your kata.
```